File: C# Actividad POO/TestApp/ParqueApp/Atraccion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ParqueApp {
    public class Atraccion {
        public string Nombre { get; set; }
        public string Tipo { get; set; }
        public double Precio { get; set; }

        public Atraccion(string nombre, string tipo, double precio) {
            Nombre = nombre;
            Tipo = tipo;
            Precio = precio;
        }

        public Atraccion()
        {
            Nombre = "Atracción Desconocida";
            Tipo = "Tipo Desconocido";
            Precio = 0.0;
        }


        public override string ToString() {
            return $"Atraccion(Nombre = { Nombre}, Tipo = { Tipo}, Precio = { Precio})";
        }
    }
}

File: ParquePY/atraccion.py
class Atraccion:
    def __init__(self, nombre="Atracción Desconocida", tipo="Tipo Desconocido", precio=0.0):
        self.nombre = nombre
        self.tipo = tipo
        self.precio = precio

    def __str__(self):
        return f"Atraccion(Nombre={self.nombre}, Tipo={self.tipo}, Precio={self.precio})"
